examples/mixture/plot_dirichlet_process_mixture.py

```diff
@@ -0,0 +1,115 @@
+"""
+=============================================
+Dirichlet Process Mixture Beta Prior Analysis
+=============================================
+
+Plot the resulting ellipsoids of a mixture of three Gaussians with
+a Dirichlet Process Gaussian Mixture for three different values of the
+beta concentration prior.
+
+For all models, the Dirichlet Process Gaussian Mixture adapts its number of
+mixture automatically. The parameter `beta_concentration_prior` has a
+direct link with the resulting number of components. Specifying a high value of
+`beta_concentration_prior` leads more often to uniformly-sized mixture
+components, while specifying small values will lead to some mixture
+components getting almost all the points while most mixture components will be
```

Comment:
will lead to some mixture components getting almost all the points while most of the other mixture components centering on just a few of the remaining points.